src/components/vertical-menu/responsive-vertical-menu/responsive-vertical-menu-divider/responsive-vertical-menu-divider.component.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { MarginProps } from "styled-system";
+
+import {
+  StyledResponsiveVerticalMenuDivider,
+  StyledHr,
+} from "./responsive-vertical-menu-divider.style";
+import { filterStyledSystemMarginProps } from "../../../../style/utils";
+import { useDepth } from "../__internal__/depth.context";
+import { useResponsiveVerticalMenu } from "../responsive-vertical-menu.context";
+
+export interface ResponsiveVerticalMenuDividerProps extends MarginProps {}
+
+export const ResponsiveVerticalMenuDivider = ({
```

Comment:
nitpick (non-blocking):

```suggestion
export const ResponsiveVerticalMenuDivider = (props) => 

...

<StyledHr {...filterStyledSystemMarginProps(props)} />
```